dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNHvdcLineInterfaceIIDM.cpp

```diff
@@ -155,8 +155,13 @@ void HvdcLineInterfaceIIDM::importStaticParameters() {
   staticParameters_.clear();
 
   bool isACEmulationEnabled = hvdcActivePowerControl_ && hvdcActivePowerControl_->isEnabled() && hvdcActivePowerControl_->isEnabled().get();
+  double p0ACEmulation = !isACEmulationEnabled ? 0 : hvdcActivePowerControl_->getP0().get();
+  double p0ACEmulationPu = (hvdcLineIIDM_.getConvertersMode() == powsybl::iidm::HvdcLine::ConvertersMode::SIDE_1_RECTIFIER_SIDE_2_INVERTER) ?
+                           p0ACEmulation / SNREF : - p0ACEmulation / SNREF;
   staticParameters_.insert(std::make_pair("isACEmulationEnabled", StaticParameter("isACEmulationEnabled",
                              StaticParameter::BOOL).setValue(isACEmulationEnabled)));
+  staticParameters_.insert(std::make_pair("p0ACEmulationPu", StaticParameter("p0ACEmulationPu",
+                             StaticParameter::DOUBLE).setValue(p0ACEmulationPu)));
```

Comment:
I don't really like having the sign change hidden into the pu transformation. I would suggest an alternative implementation, that I find clearer:

double p0ACEmulationPu = 0;
if (isACEmulationEnabled) {
  double p0ACEmulation = (hvdcLineIIDM_.getConvertersMode() == ) .. hdvcActivePowerControl ... : - hvdcActivePowerControl...
  p0ACEmulationEnabledPu = p0ACEmulation / SNREF;
}

Reply:
Ok, thanks